.github/workflows/ci.yaml

```diff
@@ -389,3 +389,29 @@ jobs:
       - run: ./devel/create-github-release "${{github.ref_name}}" dist/* nextstrain-cli-*-standalone-*
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  report-failure:
+    if: ${{ always() && github.ref_name == github.event.repository.default_branch }}
+    needs:
+      - lint
+      - test-source
+      - build-dist
+      - build-standalone
+      - test-dist
+      - test-standalone
+      - doc
+      - release
```

Comment:
_non-blocking_

It feels like it's going to be easy to forget to update this if we add new jobs. :/